### ISSUE TITLE
Make windows install script more user friendly.

### ISSUE THREAD

**Issue by jazzybluesea** [openenclave/openenclave#2141]
Currently the script outputs a lot of noise.  If there is a bug in the script--which I have encountered many times--it is very very hard to spot.

**Goals:**
1. Move the detail outputs to a file.
2. Display milestone installations in green.
3. Display failures in red.
4. Display overall success/failure of the installation.

**Comment by yakman2020** [comment]
Pls file a PR on this.

**Comment by radhikaj** [comment]
@BRMcLaren Please check if this can be folded into the work you are doing.

**Comment by BRMcLaren** [comment]
@radhikaj No, this cannot be folded into the current work. The install script primarily focuses on an end-user compiling from source whereas the work I am currently doing is for cloud CI/CD.

They are disjoint, unfortunately.
